File: test/fixtures/output/ruby/native/custom-method.rb
require 'uri'
require 'net/http'
require 'openssl'

class Net::HTTP::Propfind < Net::HTTPRequest
  METHOD = 'PROPFIND'
  REQUEST_HAS_BODY = 'false'
  RESPONSE_HAS_BODY = true
end

url = URI("https://httpbin.org/anything")

http = Net::HTTP.new(url.host, url.port)
http.use_ssl = true

request = Net::HTTP::Propfind.new(url)

response = http.request(request)
puts response.read_body

File: __tests__/__fixtures__/output/node/axios/custom-method.js
const axios = require("axios").default;

const options = {method: 'PROPFIND', url: 'https://httpbin.org/anything'};

axios.request(options).then(function (response) {
  console.log(response.data);
}).catch(function (error) {
  console.error(error);
});
